test/entrypoint/srv_with_hotreload.lua

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env tarantool
+
+require('strict').on()
+
+local log = require('log')
+local errors = require('errors')
+local cartridge = require('cartridge')
+errors.set_deprecation_handler(function(err)
+    log.error('%s', err)
+    os.exit(1)
+end)
+
+local ok, err = errors.pcall('CartridgeCfgError', cartridge.cfg, {
+    roles = {
+        'cartridge.roles.metrics',
+    },
+    roles_reload_allowed = true,
+})
+if not ok then
+    log.error('%s', err)
+    os.exit(1)
+end
```

Comment:
Please, supplement it with default `set_export` routes (and test it)

Reply:
P.S. Now it raises the error: "metrics.lua:122: attempt to index field 'current_paths' (a nil value)"